verson 4.5.0 dosen't suport npm5.6.0

### Version
4.5.0







### Environment info
```
npm 5.6.0
node 8.11.3
```


### Steps to reproduce
in npm 5.6.0

execute `npm install` with @vue/cli-service v4.5.0 

get error:

error in npm install
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "npm:": npm:vue-loader@^16.0.0-beta.3

### What is expected?
npm install can be executed in npm5.6.0

### What is actually happening?
@vue/cli-service v4.5.0 dosen't support npm 5.6.0

---
Notice that package aliasing is support since npm 6.9.0

Will npm5.6.0 be supported again?

<!-- generated by vue-issues. DO NOT REMOVE -->

--- a/packages/@vue/cli-service/package.json
+++ b/packages/@vue/cli-service/package.json
@@ -72,13 +72,15 @@
     "thread-loader": "^2.1.3",
     "url-loader": "^2.2.0",
     "vue-loader": "^15.9.2",
-    "vue-loader-v16": "npm:vue-loader@^16.0.0-beta.3",
     "vue-style-loader": "^4.1.2",
     "webpack": "^4.0.0",
     "webpack-bundle-analyzer": "^3.8.0",
     "webpack-chain": "^6.4.0",
     "webpack-dev-server": "^3.11.0",
     "webpack-merge": "^4.2.2"
+  },
+  "optionalDependencies": {
+    "vue-loader-v16": "npm:vue-loader@^16.0.0-beta.3"
   },
   "peerDependencies": {
     "@vue/compiler-sfc": "^3.0.0-beta.14",

